README.md

```diff
@@ -224,13 +123,136 @@ You'll just have to make sure that your app doesn't have any tweens that do this
 Tweens should encode any redirect locations that they generate,
 [like this](https://github.com/hypothesis/pyramid-sanity/blob/d8492620225ec6be0ba28b3eb49d329ef1e11dc2/src/pyramid_sanity/_egress.py#L22-L30).
 
+Error details
+-------------
+
+If you would like to reproduce the errors an [example app](#addendum-example-application) is given at the end
+of this section. All of the presented `curl` commands work with this app.
+
+### Badly encoded query parameters makes `request.GET` crash
+
+```terminal
+curl 'http://localhost:6543/foo?q=%FC'
+```
+
+**By default**
+
+WebOb raises `UnicodeDecodeError`. As there is no built-in exception view for
+this exception the app crashes.
+
+**With `pyramid-sanity`**
+
+A `pyramid_sanity.exceptions.InvalidQueryString` is returned which results in a
+`400: Bad Request` response.
```

Comment:
These sections are intended that they should sort of read like a sentence from "By default..."